pkg/apimachinery/identity/namespace.go

```diff
@@ -15,6 +15,7 @@ const (
 	NamespaceAnonymous      Namespace = "anonymous"
 	NamespaceRenderService  Namespace = "render"
 	NamespaceAccessPolicy   Namespace = "access-policy"
+	NamespaceProvisioning   Namespace = "provisioning"
```

Comment:
@kalleep -- this is so we can identify the user that runs provisioning.  In SQL it is currently internal ID -1, but we know that only happens with provisioning 🤷🏻